src/components/presentational/SingleSurveyTask/SingleSurveyTask.css

```diff
@@ -62,3 +63,13 @@
     {
         cursor: pointer;
     }
+
+.button{
```

Comment:
I would leave the button width inconsistent like it is today; we have now had it designed both ways so unless it was really intended that button width be consistent here we should settle on one approach and I inclined to do the one that does not require changes.  I think the different width actually serves as a nice visual indicator that this row is different.  So I would revert this change